frontend/src/modules/Environments/components/EnvironmentMLStudio.js

```diff
@@ -0,0 +1,168 @@
+            <Grid item lg={8} xl={9} xs={12}>
+              <Card>
+                <CardHeader title="ML Studio Information" />
+                <Divider />
+                <CardContent>
+                  <Typography color="textSecondary" variant="subtitle2">
+                    SageMaker ML Studio Domain Name
+                  </Typography>
+                  <Typography color="textPrimary" variant="body2">
+                    {mlStudioDomain.sagemakerStudioDomainName}
+                  </Typography>
+                </CardContent>
+                <CardContent>
+                  <Typography color="textSecondary" variant="subtitle2">
+                    SageMaker ML Studio Default Execution Role
+                  </Typography>
+                  <Typography color="textPrimary" variant="body2">
+                    arn:aws:iam::{environment.AwsAccountId}:role/
+                    {mlStudioDomain.DefaultDomainRoleName}
+                  </Typography>
+                </CardContent>
+                <CardContent>
+                  <Typography color="textSecondary" variant="subtitle2">
+                    Domain VPC Type
+                  </Typography>
+                  <Typography color="textPrimary" variant="body2">
+                    {mlStudioDomain.vpcType}
+                  </Typography>
+                </CardContent>
+                {mlStudioDomain.vpcType === 'imported' && (
+                  <>
+                    <CardContent>
+                      <Typography color="textSecondary" variant="subtitle2">
+                        Domain VPC Id
+                      </Typography>
+                      <Typography color="textPrimary" variant="body2">
+                        {mlStudioDomain.vpcId}
+                      </Typography>
+                    </CardContent>
+                    <CardContent>
+                      <Typography color="textSecondary" variant="subtitle2">
+                        Domain Subnet Ids
+                      </Typography>
+                      <Typography color="textPrimary" variant="body2">
+                        {mlStudioDomain.subnetIds?.map((subnet) => (
+                          <Chip
+                            sx={{ mr: 0.5, mb: 0.5 }}
+                            key={subnet}
+                            label={subnet}
+                            variant="outlined"
+                          />
+                        ))}
+                      </Typography>
+                    </CardContent>
+                  </>
+                )}
+              </Card>
+            </Grid>
+            <Grid item lg={4} xl={3} xs={12}>
+              <ObjectMetadata
+                accountId={environment.AwsAccountId}
+                region={environment.region}
+                environment={environment}
+                owner={mlStudioDomain.owner}
+                created={mlStudioDomain.created}
+              />
+            </Grid>
+          </Grid>
```

Comment:
I was just wanting to add all relevant information we could about domain - however the `ObjectMetadata` is esentially the same as the `ObjectMetadata` of the Environment in the `Overview` Tab - so I can remove and just keep ML Studio Information if you think that is cleaner from UX side?

Reply:
I would remove it, first because it might lead to errors on users thinking that the domain was created at the environment creation time. And also because we need to do some tweaking for it to look good on all screen sizes